modules/ROOT/pages/performance-and-impact.adoc

```diff
@@ -10,11 +10,61 @@ Based on typical usage, the expected CPU utilization is less than 5%. For more i
 
 Anypoint Monitoring asynchronously collects data, which does not block Mule runtime data processing.
 
+=== Gold and Platinum Subscriptions
+
+Gold and Platinum subscribers to Anypoint Platform receive modest data retention and feature limits:
+
+Logs::
+Log limits are documented in xref:runtime-manager::viewing-log-data.adoc#log-persistence[Runtime Manager].
+
+Application Metrics::
+30 days
+
+Enhanced API Metrics::
+30 days. The limit for data in the *Client Applications* tab is 1 hour.
+
+Basic Alerts::
+50 basic alerts
+
+////
+Gold and Platinum subscribers to Anypoint Platform receive
 * Minute Resolution for Metrics.
 * Gold and Platinum subscriptions:
  ** API analytics metrics data are retained for 90 days.
  ** Application metrics data are retained for 30 days.
+////
+
+=== Titanium Subscriptions
+
+Titanium subscribers to Anypoint Platform receive more robust data retention and feature limits:
+
+Logs::
+In raw format, your log limit is equal to the storage space that you receive with your Titanium subscription.
+// do we have a link for this...?
+
+Searchable logs::
+10% of purchased storage is searchable by size, but not by day. For example, you can use up 10% for 5 days of logs or 25 days. You can also download logs.
```

Comment:
This reads a bit funny, and we should provide more context regarding what this is about. Suggestion: "Anypoint Monitoring creates indexes on raw logs in order to allow customers to efficiently and quickly perform log search queries. We index the most recent logs for searching, up to the 10% of the total log size. To access logs that are older, they can be downloaded. See ......"

Reply:
10% is size limit for searchable logs - note index limit?

Reply:
```suggestion
10% of purchased storage can be allocated for searchable logs. +
Anypoint Monitoring creates indexes on raw logs to enable you to efficiently and quickly perform log search queries. Anypoint Monitoring indexes the most recent logs for searching, up to 10% of the total log size. You can download logs to access older logs.
```

Reply:
“10% of the total log size” has caused some confusion before. “10% of the total purchased storage size” is probably better.